src/sql/engine/px/ob_dfo.cpp

```diff
@@ -29,10 +29,13 @@ OB_SERIALIZE_MEMBER(ObP2PDhMapInfo,
                     p2p_sequence_ids_,
                     target_addrs_);
 
+OB_SERIALIZE_MEMBER(ObQcRecordData, sql, tid);
+
 OB_SERIALIZE_MEMBER(ObPxSqcMeta,
                     execution_id_,
                     qc_id_,
                     dfo_id_,
+                    record_data_,
```

Comment:
Need to consider compatible problem, so u should put this variable on the end when serializing the structure.

Reply:
> Need to consider compatible problem, so u should put this variable on the end when serializing the structure.

done